fe/fe-core/src/main/java/com/starrocks/credential/hdfs/HDFSCloudCredential.java

```diff
@@ -104,6 +103,13 @@ public FileStoreInfo toFileStoreInfo() {
         FileStoreInfo.Builder fileStore = FileStoreInfo.newBuilder();
         fileStore.setFsType(FileStoreType.HDFS);
         HDFSFileStoreInfo.Builder hdfsFileStoreInfo = HDFSFileStoreInfo.newBuilder();
+        if (!authentication.isEmpty()) {
```

Comment:
if authentication is provided, it is must be 'simple' or 'kerberos', looks like this condition is not checked.